libfreerdp/codec/rfx_rlgr.c

```diff
@@ -80,7 +80,9 @@ static INIT_ONCE rfx_rlgr_init_once = INIT_ONCE_STATIC_INIT;
 
 static BOOL CALLBACK rfx_rlgr_init(PINIT_ONCE once, PVOID param, PVOID *context)
 {
+#ifndef __MINGW32__
```

Comment:
Why this? If the symbol is not native, it should be replaced by `WinPR` version of it.